### PR TITLE
CI for Vercel

### PR DESCRIPTION
**Pull request by gooseops**
[comment]: <> (don't forget to run `yarn format` from the root :D)
Added CI to deploy to Vercel

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   ],
   "scripts": {
     "postinstall": "patch-package",
-    "build": "turbo run build --continue --filter=\"!@dao-dao/storybook\"",
+    "build:original": "turbo run build --continue --filter=\"!@dao-dao/storybook\"",
```
Comment (magiodev) [inline package.json]: I added this on debugging stage, i dont know if we want to preserve this or just get rid of build, renaming build:original back to build.

**Review by magiodev — COMMENTED** [review]
Looks good! Just a minor comment

File: package.json (hunk 1 above)
Comment (gooseops) [reply package.json]: I have no opinion on this.  It works as it is right now.  I suspect that decision would be a developer decision and not mine to make.

Comment (gooseops) [reply package.json]: @akure Do you have an opinion on this?

**Review by magiodev — APPROVED** [review]
looks good!

**Pull request merged**